packages/ma-components/local/focus-indicator/focus-indicator.css

```diff
@@ -0,0 +1,42 @@
+.ma-focus-indicator {
+  --ma-focus-indicator-background: var(--basis-focus-background-color);
```

Comment:
Voor het nageslacht zou het handig zijn als er een comment is die uitlegt:
- dit is voor obscure Safari versie X, omdat usage Y nog boven minimum percentage Z is
- hier is een linkje naar de blog post die uitlegt wat dit voor `:focus-visible` fallback constructie is